pkg/operation/botanist/logging.go

```diff
@@ -39,9 +50,18 @@ func (b *Botanist) DeploySeedLogging(ctx context.Context) error {
 		return err
 	}
 
+	hvpaValues := make(map[string]interface{})
+	hvpaEnabled := gardenletfeatures.FeatureGate.Enabled(features.HVPA)
+	if b.ShootedSeed != nil {
+		hvpaEnabled = gardenletfeatures.FeatureGate.Enabled(features.HVPAForShootedSeed)
```

Comment:
Should we keep the VPA in case the HVPA feature gates are disabled?

Reply:
In my opinion - No. With the current behaviour with these evictions, I think the Loki will work better without the VPA

Reply:
OK, sure, probably this makes sense.